NEWS.md

```diff
@@ -1,5 +1,5 @@
 # Version 1.0.9000.9098
-* Renamed `stopList` slot in `StoppingList` class.
+* Renamed `stopList` slot in `StoppingList` and `StoppingAll` classes.
```

Comment:
I think it would be good if those detailed updates would be moved to a section in the migration vignette instead of being in the news. otherwise news gets too long. can do that as separate issue